src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -24,5 +25,7 @@ public interface IProjectContext
         IEnumerable<string> CompilationItems { get; }
         IEnumerable<string> EmbededItems { get; }
         string FindProperty(string propertyName);
+        IEnumerable<DependencyDescription> Dependencies { get;}
```

Comment:
I think renaming the property here would make more sense. 
`DesignTimePackageDependencies` IMO would be more apt here.